newIDE/app/src/Utils/GDevelopServices/Game.js

```diff
@@ -154,25 +183,25 @@ export const getAclsFromUserIds = (
     level: 'owner',
   }));
 
-export const listAllShowcasedGames = (): Promise<AllShowcasedGames> => {
-  return axios
-    .get(`${GDevelopGameApi.baseUrl}/showcased-game`)
-    .then(response => response.data)
-    .then(({ gamesShowcaseUrl, filtersUrl }) => {
-      if (!gamesShowcaseUrl || !filtersUrl) {
-        throw new Error('Unexpected response from the resource endpoint.');
-      }
-      return Promise.all([
-        axios.get(gamesShowcaseUrl).then(response => response.data),
-        axios.get(filtersUrl).then(response => response.data),
-      ]).then(([showcasedGames, filters]) => ({
-        showcasedGames,
-        filters,
-      }));
-    });
+export const listAllShowcasedGames = async (): Promise<AllShowcasedGames> => {
+  const response = await axios.get(`${GDevelopGameApi.baseUrl}/showcased-game`);
+  const { gamesShowcaseUrl, filtersUrl } = response.data;
+  if (!gamesShowcaseUrl || !filtersUrl) {
+    throw new Error('Unexpected response from the resource endpoint.');
+  }
+
+  const [showcasedGames, filters] = await Promise.all([
```

Comment:
Could you check `listAllPublicAssets` to apply the same logic with the errors? I did the change on this method because if some of the API calls in the promise errors, the error doesn't bubble up unless you add the `catch(e => e)`